packages/gatsby-source-drupal/src/normalize.js

```diff
@@ -64,7 +64,7 @@ exports.downloadFile = async (
       // Ignore
     }
     if (fileNode) {
-      node.localFile___NODE = fileNode.id
```

Comment:
@freiksenet is there a performance difference between using schema customization with @link vs field___NODE or are they just different ways of accomplishing the same thing?

Reply:
If type inference is enabled - there is no difference in terms of performance. But if you use schema customization AND disable inference for a `node` type, then you can get a significant performance boost

Reply:
Makes sense, thanks @vladar!